framework/src/utils/PerfGraph.C

```diff
@@ -74,7 +75,10 @@ PerfGraph::getNumCalls(const std::string & section_name)
   auto section_it = _section_time.find(section_name);
 
   if (section_it == _section_time.end())
-    mooseError("Unknown section_name: ", section_name, " in PerfGraph::getNumCalls() ");
+    mooseError(
+        "Unknown section_name: ",
+        section_name,
+        " in PerfGraph::getNumCalls()\nIf you are attempting to retrieve the root use \"Root\".");
```

Comment:
This seems unnecessary.  The name you use is always what shows up in the output.  There are a million options but all you need to do is put exactly what's in the output an you're good.